depends/hosts/darwin.mk

```diff
@@ -0,0 +1,17 @@
+OSX_MIN_VERSION=10.7
+OSX_SDK_VERSION=10.9
+OSX_SDK=$(SDK_PATH)/MacOSX$(OSX_SDK_VERSION).sdk
+LD64_VERSION=241.9
```

Comment:
(This is more of a general reference note.)

"ld -v" will yield the ld64 version, although Wikipedia sometimes lists the version. For Xcode 6.2, this value is accurate. Last I checked, Xcode 6.3 is 242, although this may change before the final version comes out.